### ISSUE TITLE
Mime Types (File) Pdf

### ISSUE THREAD

**Issue by haakym** [proengsoft/laravel-jsvalidation#91]
Firstly, amazing package - thank you for your contribution!

Having an issue with this rule: http://laravel.com/docs/5.1/validation#rule-mimes

If I have a file upload in my form, set the rules to require PDF only, like so:

```php
'attachment' => 'mimes:pdf',
```

then upload a jpeg file and submit the form I get `string 'The attachment must be a file of type: pdf.' (length=43)` when running

```php
var_dump($errors->first())
```

however nothing seems to be triggered from the javascript validation side. If this isn't a feature what would be the correct way to go around this? Would I just need to add in `has-error` to the `form-group` of the file input with the respective error?

**Comment by haakym** [comment]
This seems to work for now:

```php
<div class="form-group {{ $errors->first('attachment', 'has-error') }}">
                    <label for="attachment">Test File:</label>
                    <input type="file" name="attachment" id="attachment" class="form-control" value="{{ Input::old('attachment') }}">
                    {!! $errors->first('attachment', '<span id="attachment-error" class="help-block error-help-block">:message</span>') !!}
                </div>
```

**Comment by torrentalle** [comment]
Hi,
The Javascript validation should be triggered if the field name is `attachment`. The message is shown using the [errorPlacement]
(https://github.com/proengsoft/laravel-jsvalidation/blob/master/resources/views/bootstrap.php#L8)  function. So, you can customize this function or put your field into element with `form-group` class.

Thank's for your feedback!
